### ISSUE TITLE
How to start?

### ISSUE THREAD

**Issue by SuperiorLeo** [zbzhzhy/Unsupervised-Spectral-Reconstruction#2]
It's sorry to bother you. But your work has GAN and semantic information. Can you give me more guide to start your job? I don't know how to run it.

**Comment by ZHU-Zhiyu** [comment]
Our framework is generally based on the HR-Net. To start the network training, you should prepare dataset (both the hyperspectral and semantic segmentation datasets), I'll release the hyperspectral dataset soonly. Please refer (https://github.com/HRNet/HRNet-Semantic-Segmentation/tree/pytorch-v1.1), and
(https://github.com/zhanghang1989/detail-api) for the installation of Pascal Context dataset. After the data preparation, you can train the network directly. We will give a more friendly version soonly. If any question, feel free to ask.

**Comment by xuyixuan1999** [comment]
> Our framework is generally based on the HR-Net. To start the network training, you should prepare dataset (both the hyperspectral and semantic segmentation datasets), I'll release the hyperspectral dataset soonly. Please refer (https://github.com/HRNet/HRNet-Semantic-Segmentation/tree/pytorch-v1.1), and (https://github.com/zhanghang1989/detail-api) for the installation of Pascal Context dataset. After the data preparation, you can train the network directly. We will give a more friendly version soonly. If any question, feel free to ask.

I have prepared all the data, but how to train it? I find it locks train.py and test.py compare  the HR-Net.

**Comment by ZHU-Zhiyu** [comment]
> > Our framework is generally based on the HR-Net. To start the network training, you should prepare dataset (both the hyperspectral and semantic segmentation datasets), I'll release the hyperspectral dataset soonly. Please refer (https://github.com/HRNet/HRNet-Semantic-Segmentation/tree/pytorch-v1.1), and (https://github.com/zhanghang1989/detail-api) for the installation of Pascal Context dataset. After the data preparation, you can train the network directly. We will give a more friendly version soonly. If any question, feel free to ask.
> 
> I have prepared all the data, but how to train it? I find it locks train.py and test.py compare the HR-Net.

Hi, @xuyixuan1999  I have uploaded the trianing code. You can run the bash file to train the model. Remember to prepare the pretrained segmentation model.

**Comment by zr-bee** [comment]
> > Our framework is generally based on the HR-Net. To start the network training, you should prepare dataset (both the hyperspectral and semantic segmentation datasets), I'll release the hyperspectral dataset soonly. Please refer (https://github.com/HRNet/HRNet-Semantic-Segmentation/tree/pytorch-v1.1), and (https://github.com/zhanghang1989/detail-api) for the installation of Pascal Context dataset. After the data preparation, you can train the network directly. We will give a more friendly version soonly. If any question, feel free to ask.
> 
> I have prepared all the data, but how to train it? I find it locks train.py and test.py compare the HR-Net.

Why is my data set inconsistent with the data set mentioned in the author's experiment? For example, about the NTIRE '20 dataset and 1525 of the Cityscape datasets for the test set, did I not find the correct download method?
